components/shared/internalLinks/InternalLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { ILink } from "@kontent-ai/delivery-sdk"
+import { FC, ReactNode } from "react";
+import { contentTypes } from "../../../models";
+import { ProductLink } from "./ProductLink";
+
+type Props = Readonly<{
+  link: ILink;
+  children: ReactNode;
+}>;
+
+export const InternalLink: FC<Props> = props => {
```

Comment:
nitpick, destructure object

Reply:
In my opinion, this is better than destructuring props, because it is visible, in the body, where the values come from (the props) and the `props.` prefix isn't that long anyway.